db/migrate/20211213165628_change_param_null_in_fields_to_adresses.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeParamNullInFieldsToAdresses < ActiveRecord::Migration[6.1]
+  def change
+    change_table :adresses do |t|
+      t.change :cep, :bigint, :null => false
```

Comment:
Esses campos de codigos, deixaria sempre como string, pois se você usar como int, caso o primeiro digitor for zero ele será perdido:

ex: 
codigo: 012345
Ao armazenar no banco ficara: 12345

Você perderá o zero

Reply:
Ok, vou corrigir isso!